src/tests/tablet/schedule/AMeetingCannotBeUpdatedWhenItHasConflictsWithOtherMeeting.java

```diff
@@ -0,0 +1,89 @@
+import org.testng.annotations.Test;
+
+import framework.common.MeetingMethods;
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC12: Verify an existing meeting cannot be updated if it has conflicts with another meeting
+ * with another meeting
+ * @author Asael Calizaya
+ *
+ */
+public class AMeetingCannotBeUpdatedWhenItHasConflictsWithOtherMeeting {
+	private ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	private List<Map<String, String>> meetingData = excelReader.getMapValues("MeetingData");
+	private String nameMeeting1 = meetingData.get(0).get("Subject");
+	private String organizer1 = meetingData.get(0).get("Organizer");
+	private String password1 = meetingData.get(0).get("Password");
+	
+	private String nameMeeting2 = meetingData.get(1).get("Subject");
+	private String organizer2 = meetingData.get(1).get("Organizer");	
+	private String password2 = meetingData.get(1).get("Password");
+	
+	private String roomName = meetingData.get(1).get("Room");
+	private String authentication1 = organizer1 + ":" + password1;
+	private String authentication2 = organizer2 + ":" + password2;
+
+	@BeforeMethod(groups = "ACCEPTANCE")
+	public void creationMeetingsPreCondition() {
+		MeetingMethods preCondition = new MeetingMethods();
+
+		//Meeting 01
+		preCondition.createMeetingFromHome(meetingData.get(0).get("Organizer"),
+				meetingData.get(0).get("Subject"), meetingData.get(0).get("Start time"),
+				meetingData.get(0).get("End time"), meetingData.get(0).get("Attendee"),
+				meetingData.get(0).get("Body"), meetingData.get(0).get("Password"));
+
+		//Meeting 02
+		preCondition.createMeetingFromHome(meetingData.get(1).get("Organizer"),
+				meetingData.get(1).get("Subject"), meetingData.get(1).get("Start time"),
+				meetingData.get(1).get("End time"), meetingData.get(1).get("Attendee"),
+				meetingData.get(1).get("Body"), meetingData.get(1).get("Password"));
+	}
+
+	@Test(groups = "ACCEPTANCE")
+	public void testAMeetingCannotBeUpdatedWhenItHasConflictsWithOtherMeeting() {
+		String subject = meetingData.get(2).get("Subject");
+		String startTime = meetingData.get(2).get("Start time");
+		String endTime = meetingData.get(2).get("End time");
+		String attendee = meetingData.get(2).get("Attendee");
+		String body = meetingData.get(2).get("Body");
+		String password = meetingData.get(2).get("Password");
+		
+		HomeTabletPage homeTabletPage = new HomeTabletPage();
+		SchedulePage schedulePage = homeTabletPage.clickScheduleBtn();
+		schedulePage
+		.clickOverMeetingCreated(nameMeeting1)
+		.setSubjectTxtBox(subject)
+		.setStartTimeDate(startTime)
+		.setEndTimeDate(endTime)
+		.setAttendeeTxtBoxPressingEnter(attendee)
+		.setBodyTxtBox(body)
+		.clickUpdateBtn()
+		.confirmCredentials(password);
+
+		//Fails because the error message is not related with the error
```

Comment:
Fails, because the error message displayed is not related to the problem, it should be more clear